skelebot/systems/scaffolding/scaffolder.py

```diff
@@ -47,18 +64,31 @@ def scaffold(self):
 
         # Configure Template Variables
         self.variables["name"] = name
-        self.variables["name_simple"] = name.lower().replace(" ", "_")
+        self.variables["name_simple"] = name.lower().replace(" ", "_").replace("-", "_")
         self.variables["description"] = description
         self.variables["maintainer"] = maintainer
         self.variables["contact"] = contact
         self.variables["language"] = language
 
         # Load the Template Config
-        template = promptUser("Select a TEMPLATE", options=list(TEMPLATES[language].keys()))
-        template_name = TEMPLATES[language][template]
-        template_path = TEMPLATE_PATH.format(name=template_name)
-        template_path = os.path.join(os.path.dirname(__file__), template_path)
-        template = self.__load_template(template_path) 
+        options = list(TEMPLATES[language].keys())
+        options.append("github:{owner}:{repo}:{branch}")
+        template = promptUser("Select a TEMPLATE", options=options, reiterate=False)
```

Comment:
I think, with reiterate=False, if the user makes a typo, it will throw an error instead of asking again.

Reply:
ya, with the refactor to the prompts, the reiterate won't be needed. I didn't like that implementation at all